src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/Objection.java

```diff
@@ -191,4 +192,10 @@ public LocalDateTime getStatusChangedOn() {
     public void setStatusChangedOn(LocalDateTime statusChangedOn) {
         this.statusChangedOn = statusChangedOn;
     }
+    
+    public boolean isDataEnteredByUserIncomplete() {
+        return StringUtils.isEmpty(reason)
```

Comment:
I only implemented a check for the user entered data being incomplete here (since logic in web and API code should already mean the other fields can’t be empty) but maybe we should just check that the entire objection data is as we expect at this point?

Reply:
I think reason is the only field that could ever be empty here. It seems strange to check come mandatory fields but not others unless we can think of a possible edge case where they would be empty.

Reply:
Ignore this I can see how these might be null now.